### ISSUE TITLE
Creating or editing presets limited to 2 cells

### ISSUE THREAD

**Issue by pH7-JP1** [johncclayton/electric#127]
Presets created in the charger with more than 2 cells show a blank when editing the preset and only "Auto", 1, and 2 are available in the pulldown. Same when creating a new preset with the app, both Android and iOS. 

Edit 1: I guess I should have made it clear - the problem is that I can't create a new preset with a cell count larger than 2. This also effects the preset that is automatically created if you use the "Capacity & C" option for charging. I connected a 3-cell battery and used this option. The charger raised an error - something like, "Wrong number of cells". The automatically created preset had only 2 cells.

Edit 2: The same problem exists for the automatically created preset for "Measure IR" it is always 2-cell just like the one created for "Capacity & C".

**Comment by pH7-JP1** [comment]
7 days and no comment?? Either no one is watching the issues, or no one believes this issue. In case it is the later, I created a video: [https://youtu.be/fh0Y25Fm1HM](url)

At the beginning of the video you can see that there is no current "Electric Charge Plan" preset. Then I start a charge for 1 5000mah battery at 2C. You can see at the beginning that a 6-cell battery is attached. The error "Number of cells doesn't match the setting" pops up on the charger. You can see why by looking at the Electric Charge Plan that was automatically created - It is for 7.4V (2S).

**Comment by pH7-JP1** [comment]
Yep, I understand that. I at least expected that someone else, not necessarily one of the developers, would comment that they had observed the same problem. The video that is part of the readme on the main github page shows that this did work correctly at one time. 

I wrote software myself until I retired 14 years ago. I may look into the problem myself, but for someone who knows their way around this development environment this may be a few minute fix while it could take me many days just to "learn to walk". 

Thanks for your response though. It at least confirms that other people do read the issues here. I wasn't even sure that I posted in the correct place.
